doc/examples.md

```diff
@@ -1,13 +1,69 @@
 # Usage examples
 
-- [Instrument output schemas](#instrument-output-schemas)
+The data model library provides access to different sequencer and bioinformatic pipeline output schemas
+in JSON that have been defined in collaboration with the labs.
 
-## Instrument output schemas
+Overview:
 
-The data model library provides access to different sequencer output schemas
-in JSON that have been defined in collaboration with the labs.
+- [Oxford Nanopore output](#oxford-nanopore-output)
+
+    * [Create Nanopore experiment object](#create-nanopore-experiment-object)
+    * [Validate Nanopore data structure](#validate-nanopore-data-structure)
+    * [Nanopore experiment object implementation example](#nanopore-experiment-object-implementation-example)
+    
+- [Nf Core pipeline output](#data-models)
+
+    * [Create NfCore experiment object](#create-nfcore-experiment-object)
+    * [Validate NF-Core data structure](#validate-nf-core-data-structure)
+    * [NfCore experiment object implementation example](#nf-core-experiment-object-implementation-example)
+
```

Comment:
Do we need both examples? Shouldn't they be similar in usage? I would prefer to only have on example if they are used the same (what I would expect)